crates/noirc_evaluator/src/ssa_refactor/ir/basic_block.rs

```diff
@@ -11,7 +11,7 @@ use super::{
 /// This means that if one instruction is executed in a basic
 /// block, then all instructions are executed. ie single-entry single-exit.
 #[derive(Debug, PartialEq, Eq, Hash, Clone)]
-pub(crate) struct BasicBlock {
```

Comment:
@jfecher do we want to make these public?

Reply:
It could stay `pub(crate)`, it looks like it isn't used outside the same crate still